src/plugins/validation/2and3/semantic-validators/schema-ibm.js

```diff
@@ -617,23 +617,26 @@ function checkProperties(
             messages.addMessage(
               propertiesToCompare[key].path,
               `Property has inconsistent type: ${key}.`,
-              configOption,
+              configOption[0],
               'inconsistent_property_type'
             );
           }
           messages.addMessage(
             contextPath.concat(['properties', key]).join('.'),
             `Property has inconsistent type: ${key}.`,
-            configOption,
+            configOption[0],
             'inconsistent_property_type'
           );
         }
       } else {
-        propertiesToCompare[key] = {
-          type: value.type,
-          path: contextPath.concat(['properties', key]).join('.'),
-          printed: false
-        };
+        if (configOption && !configOption.includes(key)) {
+          // add property if the name is not generic
```

Comment:
This is really picky 😅, but users may exclude non-generic properties

```suggestion
          // add property if the name is not excluded
```